src/BuildOnWindows.cmake

```diff
@@ -178,6 +178,7 @@ foreach(lib ${TORCH_XPU_OPS_LIBRARIES})
   torch_compile_options(${lib})
   target_compile_options_if_supported(${lib} "-Wno-deprecated-copy")
   target_compile_options(${lib} PRIVATE ${TORCH_XPU_OPS_FLAGS})
+  target_compile_options(${lib} PRIVATE /permissive-)
```

Comment:
Could you elaborate the logic of the option, like What new C++ feature is brought by 2025.0 compiler? how the option fixed the build error with 2025.0 compiler?